.eslintrc

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": [
+    "@nuxtjs/eslint-config-typescript"
+  ],
+  "rules": {
+    "no-use-before-define": ["off"],
+    "@typescript-eslint/no-use-before-define": ["error", {
+      "functions": false,
+      "classes": false,
+      "variables": false,
+      "typedefs": true
+    }],
```

Comment:
@danielroe 

IMHO, it's convenient to have `"typedefs": true` defined in `@nuxtjs/eslint-config-typescript`, because it would be common to refer to other types before their definitions.

What do you think?

(note: the other 3 flags are copied from the output of `yarn eslint --print-config ./src/module.ts`, so I believe they are defined in somewhere in the extended configs.)